pwncat/platform/windows.py

```diff
@@ -20,7 +20,6 @@
 import time
 import base64
 import shutil
-import signal
```

Comment:
This had the `flake8` warning.
I am adding this comment, because I am supposed to :thinking: ? - as per the __PR template__

Reply:
I appreciate it! That must have slipped through the cracks elsewhere. Probably my fault :tired_face: hahaha

Reply:
I saw one of your commits about re-adding an import (__readline__ module) because it was accidentally removed during `flake8` cleanup, perhaps you have something in mind for this import